packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -711,6 +758,22 @@ contract L1StandardBridge_FinalizeBridgeETH_Test is Bridge_Initializer {
 
         l1StandardBridge.finalizeBridgeETH{ value: 100 }(alice, alice, 100, hex"");
     }
+
+    /// @dev Tests that finalizing bridged ETH reverts if a custom gas token is used.
```

Comment:
This needs coverage of the `receive` function, ie `address(l1StandardBridge).call{ value: 1 ether }(hex"");`

Reply:
it should revert